# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/ncurses/tic-shlibs-run.patch.cross
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./misc/run_tic.in.orig	2004-09-21 00:08:05.106683264 +0200
+++ ./misc/run_tic.in	2004-09-21 00:08:35.937996192 +0200
@@ -66,6 +66,10 @@
 *) PATH=../progs:${DESTDIR}$bindir:$PATH ;;
 esac
 export PATH
+SHLIB=$srcdir/shlib
+else
+# Cross-compiling, so don't set PATH or run shlib.
+SHLIB=
 fi
 
 #
@@ -111,7 +115,7 @@
 	problems for older ncurses applications.
 
 EOF
-if ( $srcdir/shlib tic$suffix -s -o $TERMINFO $source )
+if ( $SHLIB tic$suffix -s -o $TERMINFO $source )
 then
 	echo '** built new '$TERMINFO
 else
